src/vdoc/models/project.py

```diff
@@ -109,6 +109,19 @@ def display_name(self) -> str:
         settings = VDocSettings()
         return settings.project_display_name_mapping.get(self.name, self.name)
 
+    @computed_field  # type: ignore[prop-decorator]  # https://docs.pydantic.dev/2.0/usage/computed_fields/
+    @cached_property
+    def category_id(self) -> Optional[int]:
```

Comment:
Ich dachte wir sind Team
```suggestion
    def category_id(self) -> int | None:
```